code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -2891,13 +2892,13 @@ var/list/bloodcult_exitportals = list()
 			vessel.my_appearance.b_eyes = 21
 			vessel.my_appearance.s_tone = 45 // super duper albino
 
+			vessel.equip_or_collect(new /obj/item/clothing/under/rags(vessel), slot_w_uniform)
```

Comment:
why is this order of operations being changed?

Reply:
I don't think it probably matters, but this was undocumented and I want to know why

Reply:
I was originally changing something else but decided not to include it in this PR. I've fixed the order of operations.